CedarJavaFFI/src/interface.rs

```diff
@@ -644,18 +681,26 @@ pub fn getEUIDRepr<'a>(
 
 fn get_euid_repr_internal<'a>(
     env: &mut JNIEnv<'a>,
-    type_name: JObject<'a>,
-    id: JObject<'a>,
+    type_name_obj: JObject<'a>,
+    id_obj: JObject<'a>,
 ) -> Result<JValueOwned<'a>> {
-    if type_name.is_null() || id.is_null() {
-        raise_npe(env)
-    } else {
-        let etype = JEntityTypeName::cast(env, type_name)?.get_rust_repr();
-        let id = JEntityId::cast(env, id)?.get_rust_repr();
-        let euid = EntityUid::from_type_name_and_id(etype, id);
-        let jstring = env.new_string(euid.to_string())?;
-        Ok(jstring.into())
+    if type_name_obj.is_null() || id_obj.is_null() {
+        return raise_npe(env);
     }
+
+    // Convert JObject -> JString
+    let type_jstr = JString::from(type_name_obj);
```

Comment:
Corresponding CedarJava method `private static native String getEUIDRepr(EntityTypeName type, EntityIdentifier id)`. I believe this would fail as the inputs are of types `EntityTypeName` and `EntityIdentifier`. Can we test this through CedarJava to verify?